integration-tests/luks/create-with-ext4.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: disk /dev/sdb with empty partition sdb1
+
+
+from sys import exit
+from storage import *
+from storageitu import *
+
+
+set_logger(get_logfile_logger())
+
+environment = Environment(False)
+
+storage = Storage(environment)
+
+staging = storage.get_staging()
+
+sdb1 = Partition.find_by_name(staging, "/dev/sdb1")
+
+luks = sdb1.create_encryption("cr_test")
```

Comment:
> The object being created is called Encryption

That does not make sense either. Encryption is a concept or an activity, not an object. When I look at the [class declaration](https://github.com/aschnell/libstorage-ng/blob/33769c90ef09168229d61a8e2f55c40217b2e7ce/storage/Devices/Encryption.h#L43) I think we should replace `Encryption` by `EncryptionLayer`.